stacker/actions/build.py

```diff
@@ -96,17 +101,19 @@ def resolve_parameters(parameters, blueprint, context, provider):
                         provider.get_output(stack_fqn, output))
                 except KeyError:
                     raise exceptions.OutputDoesNotExist(stack_fqn, v)
-            value = ",".join(v_list)
-        if value is None:
-            logger.debug("Got None value for parameter %s, not submitting it "
-                         "to cloudformation, default value should be used.",
-                         k)
-            continue
-        if isinstance(value, bool):
-            logger.debug("Converting parameter %s boolean \"%s\" to string.",
-                         k, value)
-            value = str(value).lower()
-        params[k] = value
+            elif v is None:
+                logger.debug("Got None value for parameter %s, not submitting "
+                             "it to cloudformation, default value should be "
+                             "used.", k)
+                continue
+            elif isinstance(v, bool):
+                logger.debug("Converting parameter %s boolean \"%s\" to "
+                             "string.", k, v)
+                v_list.append(str(v).lower())
+            else:
+                v_list.append(v)
+        if v_list != []:
```

Comment:
You can get rid of this if statement - if the list is empty, there will be nothing to join, and it'll result in a blank string.  Only updating the param if the list is populated is a change from the old behavior, and I'm not sure what issues it might cause.